src/Hypsography/Hypsography.jl

```diff
@@ -156,7 +160,44 @@ function ExtrudedFiniteDifferenceSpace(
     )
 end
 
-
+"""
+    diffuse_surface_elevation!(f; κ,iter,dt)
+Option for 2nd order diffusive smoothing of generated terrain.
+Compute smoothed profile before assigning the surface
+elevation to the `HypsographyAdaption` type. Laplacian 
+operator is applied with simple Euler updates to generate
+profiles for each new iteration.
+(1) Compute discrete elevation profile f
+(2) Compute diffuse_surface_elevation!(f, κ, iter). f is mutated.
+(3) Define `Hypsography.LinearAdaption(f)`
+(4) Define `ExtrudedFiniteDifferenceSpace` with new surface elevation.
+Default diffusion parameters are appropriate for spherical arrangements. 
+For `zmax-zsfc` == 𝒪(10^4), κ == 𝒪(10^8), dt == 𝒪(10⁻¹).
+Applicable on periodic domains only. 
```

Comment:
Doc formatting doesn't look correct, see https://clima.github.io/ClimaCore.jl/previews/PR1262/api/#Hypsography

You can get a live preview using https://github.com/tlienart/LiveServer.jl